robo4j-hw-rpi/src/examples/java/com/robo4j/hw/rpi/serial/ydlidar/YDLidarTest.java

```diff
@@ -55,7 +55,7 @@ public void onScan(ScanResult2D scanResult) {
 		device.shutdown();
 		System.out.println("Done!");
 		// Naughty that this has to be done... Perhaps fix Pi4J?
-		SerialFactory.shutdown();
+//		SerialFactory.shutdown();
```

Comment:
In Pi4J we are considering to remove serial in favor of jSerialComm?
https://github.com/Pi4J/pi4j-v2/discussions/308
WDYT?

Reply:
i've let it there as a reminder, as this requires to be cleaned through the next tickets. https://github.com/Robo4J/robo4j/issues/63